### ISSUE TITLE
PS4444 voltage range with PicoConnect 442

### ISSUE THREAD

**Issue by LAAL84** [colinoflynn/pico-python#195]
Hi guys,
As a newbie to Python and coding in general, and this API stuff, I just want to thank you for the great work you have done here. I have successfully connected to my scope and have tested the spectrogram function.
However, I have a problem when trying to set the voltage range for my PS4444 with HP diff. probes to more than 50 V, but I'm not allowed to do this. I tried adding the ranges in the ps4000a.py and picobase.py acc to the descriptions in the latest PicoScope 4000 Series (A API) Programmer's Guide and PicoConnect 442 Differential Probe User’s Guide, without success. I see that the existing voltage ranges is defined with a "apivalue". I can't find this for the values above 50 V. Is this required? I also see some mention of probe attenuation in the picobase.py script. The PC442 probes seems to be 25:1. Is this something that needs to be defined?

**Comment by hmaarrfk** [comment]
Yeah. You might need to tell the picoscope of your probe capabilities. Is it a 1x probe or 10x probe? Typically supplying digital electronics with more than 30V somewhat breaks sensitive transistors so you need a probe that divides the voltage by 10 or so (10x probe).
